src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/Versioning/FhirCollectionSettingsUpdater.cs

```diff
@@ -48,8 +48,6 @@ public async Task ExecuteAsync(Container container, CancellationToken cancellati
 
             if (thisVersion.Version < CollectionSettingsVersion)
             {
-                _logger.LogDebug("Ensuring indexes are up-to-date {CollectionUri}");
```

Comment:
>CollectionUri [](http://example.com/codeflow?start=67&length=13)

We don't pass anything here, and it's a debug level, so I lean towards just removing this message.